_accounts.md

```diff
@@ -111,3 +111,50 @@ Retrieves the details about a specific account.
 ### Response
 
 Returns a fully formed [Account Object](#account-object) representing the requested account.
+
+## Adding credit/debit card accounts
+
+The card details provided in this portion of the documentation are not real cards but specifically designed for Sandbox. These can be used to add card accounts to an Uphold wallet which will then allow for the testing of transactional flows.
+
+<aside class="notice">
+  Any value entered for <i>expiry date</i> and <i>CVV</i> will be accepted as valid when used in conjunction with the card data listed in this section.
+</aside>
+
+### Approved cards
+
+The following is a list of valid card data that can be used to add a card account to an Uphold wallet for testing purposes.
+
+Number           | Brand      | Type   | Country | OCT settlement |
+---------------- | ---------- | ------ | ------- | -------------- |
+5355223761921186 | mastercard | debit  | GB      | instant        |
```

Comment:
`4242424242424242` seems to be duplicated. Check also the details.

Reply:
Also the case in the document provided as the source of truth.
<img width="1254" alt="image" src="https://github.com/uphold/docs/assets/3464842/03599a55-8b8f-4da0-81ba-4a02a69a32c2">

Reply:
Same as here: https://github.com/uphold/docs/pull/241#discussion_r1455579374.